src/cmd.c

```diff
@@ -1986,6 +1985,55 @@ static void set_rfpower(atci_param_t *param)
     OK_();
 }
 
+
+static void get_recvlen(void)
+{
+    cmd_printf("$RECVLEN=%d\r\n", lrw_recv_len());
+    OK_();
+}
+
+static void get_recvget(void)
+{
+    lrw_recv_t *recv = lrw_recv_get();
+
+    log_debug("recvget: %p", (void *)recv);
+
+    if (recv == NULL)
+    {
+        cmd_print("$RECVGET=0,0,0\r\n");
+    } else {
+        cmd_printf("$RECVGET=%d,%d,%d,", lrw_recv_len(), recv->port, recv->length);
+        atci_print_buffer_as_hex(recv->buffer, recv->length);
```

Comment:
The modem has a setting which determines whether message payloads should be hex-encoded. The setting is [AT+DFORMAT](https://github.com/hardwario/lora-modem/wiki/AT-Command-Interface#atdformat). In this command you implicitly hex-encode the message, presumably to be able to read it as one line. I think this might be confusing. Now we have some AT commands that respect the AT+DFORMAT setting and some that don't.